cartridges/cloudinary_sg_changes/cartridge/templates/default/checkout/cart/cart.isml

```diff
@@ -184,7 +184,7 @@
 														style="max-width:100%"
 														alt="${lineItem.product.getImage('small',0).alt}"
 														title="${lineItem.product.getImage('small',0).title}"
-														class="cld-responsive"
+														class="${'isResponsive' in cldProductImage && cldProductImage.isResponsive ? 'cld-responsive' : ''}"
```

Comment:
This comment applies to this and all other changes about cld-responsive.
1. When using cld-responsive then no 'src' attribute should be defined, only 'data-src'
2. When not using cld-responsive then no 'data-src' attribute should be defined, only 'src'
3. Do not use "'isResponsive' in object". It is enough to check object.isResponsive

Reply:
@asad-rafter Please fix according to these. 
1. Setting the 'src' with cld-responsive is nullifying the advantage of cld-responsive, so it should be avoided
2. Setting the data-src without cld-responsive is bloating the HTML with unused attribute
3. For readability and consistency, we should use only a single way to check the attribute

Reply:
@yuval-cloudinary, these are outdated, can you please go there to https://github.com/cloudinary/cloudinary_sfcc_site_cartridge/pull/3/files and verify if your comments have not been fixed?

Reply:
Thanks, I see it now!